org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/ServerIssueTrackable.java

```diff
@@ -99,6 +99,22 @@ public String getSeverity() {
   }
 
   @Override
+  public String getRawSeverity() {
+    throw new UnsupportedOperationException();
+  }
+
+  @Override
+  public String getType() {
+    // FIXME We are not able to get server side issue type
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)